src/Elmish.WPF/Binding.fs

```diff
@@ -492,11 +492,11 @@ type Binding private () =
   /// <param name="getId">Gets a unique identifier for a collection
   /// item.</param>
   static member oneWaySeqLazy
-      (get: 'model -> 'a,
-       equals: 'a -> 'a -> bool,
-       map: 'a -> #seq<'b>,
-       itemEquals: 'b -> 'b -> bool,
-       getId: 'b -> 'id)
+      (get: 'model -> 'bindingModel,
+       equals: 'bindingModel -> 'bindingModel -> bool,
+       map: 'bindingModel -> #seq<'bindingViewModel>,
```

Comment:
I don't think `bindingViewModel` is correct.  Isn't that supposed to be either `DynamicViewModel` or some custom view model?

Reply:
Where is that?

I think it would be more consistent with the `subModel` methods to rename `a` to `subModel` and `b` to `bindingModel` (though there is also some inconsistency in those methods).

However, I am not a huge fan of those names.  All of those `subModel` methods are obsolete.  I don't want any method to "include" a call to any of the mapping functions including `alterMsgStream`.

Reply:
Compare `BindingVmHelpers.fs:98` to `BindingVmHelpers.fs:120`. It's consistent with those names.

Reply:
That is internal code. I don't think we should typically change public code to make it consistent with internal code.

Also, `bindingViewModel` is `DynamicViewModel<'a, 'b>` (for some `'a` and `'b`) there, but that is not the case for what you called `bindingViewModel` here.

Reply:
Sorry I managed to crash the GitHub phone app looking for another example this morning.

I didn't understand originally that you were just speaking about the external interface. I'll just revert this file.